### PR TITLE
Added CovidInfo Project

### PR DESCRIPTION
**Pull request by jaivsh**
## Description
I've added the covid data scrapper to the reop along withproper documentation. 
For the purpose of checking/running/executing the scrapper, I've also created a colab file here: [Colab notebook link](https://colab.research.google.com/drive/1uD11Vp1AjoBsdZJCykQGNNCEoSRknRzt?usp=sharing)

> The changes made in this pull request should be briefly described.

## Checklist

- [x] Have you adhered to the repository's defined coding convention rules?
- [x] Have you updated the 'documentation.md' file with the method/function documentation?
- [x] Have you sent a message along with the result or response?
- [x] Have you used the try-catch technique?
- [x] Has the method/class been added to the documentation (md file)?

## Screenshots
The relevant screenshots for each function are as follows: 

1. Scrape():
Screenshot 2023-07-12 at 9 12 58 PM

2. findCountry():
Screenshot 2023-07-12 at 9 14 00 PM

3. sortbycases():
Screenshot 2023-07-12 at 9 20 01 PM

4. sortbydeaths():
Screenshot 2023-07-12 at 9 21 32 PM

5. totalcases():
Screenshot 2023-07-12 at 9 22 29 PM

6. totaldeaths():
Screenshot 2023-07-12 at 9 25 39 PM

---

I certify that I have carried out the relevant checks and provided the requisite screenshot for validation by submitting this pull request.
I appreciate your contribution.

### CODE REVIEW

**Comment by jaivsh** [comment]
@nikhil25803 I've completed the implementation. PLease check out this PR

File: src/scrape_up/CovidInfo/covid_info.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+        data.append([
+            country,
+            int(confirmed),
+            int(deaths),
+            continent
+        ])
+
+      except StopIteration:
+        break
```
Comment (nikhil25803) [inline src/scrape_up/CovidInfo/covid_info.py:43]: On exception, return `None`. It helps the user to script some conditions on it.

File: src/scrape_up/CovidInfo/covid_info.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        fc.append([i[0],i[1],i[2]])
+
+        return fc
+    return 'Data not aviliable for value entered'
+
+  def sortbycases(self, rev):
+    data = self.scrape()
+    data.remove(['Country','Number of Cases', 'Deaths'])
+    data = sorted(data, key= lambda r:r[1],reverse=rev)
+    return data
```
Comment (nikhil25803) [inline src/scrape_up/CovidInfo/covid_info.py:61]: Implement try-exception in all the methods.

File: src/scrape_up/CovidInfo/covid_info.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    sol = fin.split(' ')
+    return sol[0]
+
+  def totaldeaths(self):
+    url = 'https://www.worldometers.info/coronavirus/countries-where-coronavirus-has-spread/'
+    page = requests.get(url)
+    soup = BeautifulSoup(page.text, 'html.parser')
+    req = soup.find_all('a',{"href": "/coronavirus/coronavirus-death-toll/"})
+    k = req[0].find('strong').contents[0].split(' ')
+    return k[0]
```
Comment (nikhil25803) [inline src/scrape_up/CovidInfo/covid_info.py:88]: Write a docstring for the class and also for each method

**Review by nikhil25803 — CHANGES_REQUESTED** [review]
Requested some changes @jaivsh. Implement them and update the PR.

File: src/scrape_up/CovidInfo/covid_info.py (hunk 2 above)
Comment (jaivsh) [reply src/scrape_up/CovidInfo/covid_info.py]: Sir, I've implemented the changes as you prescribed. Please review the work. Thanks!

**Comment by jaivsh** [comment]
@nikhil25803 I've performed the following changes:
- added docstring for class and functions
- added try/catch for all functions
- added return None in required places. 

Note: In first function scrape(), i initially inserted the ```except StopIteration: break``` so that when all the rows would be appended to list, and there would be no more rows, the code should run into StopIteration exception and would execute the catch block, where it would encounter break and would stop the loop. So, it is difficult to add a try/except block in first function. However, I've added it's equivalent which would ```return Null``` in case no row is added to the list.

**Review by nikhil25803 — APPROVED** [review]

**Pull request merged**